config/config.exs

```diff
@@ -29,7 +29,7 @@ config :realtime, :extensions,
   },
   postgres_cdc_stream: %{
     type: :postgres_cdc,
-    key: "postgres_cdc_stream",
+    key: "postgres_cdc_rls_stream",
```

Comment:
`postgres_cdc_stream` is actually the NON RLS version (extension) so this should not change.